src/__tests__/WeekView.test.js

```diff
@@ -1,10 +1,446 @@
-import 'react-native';
 import React from 'react';
-import renderer from 'react-test-renderer';
+import { Dimensions } from 'react-native';
+import { render, cleanup } from '@testing-library/react-native';
+import { State } from 'react-native-gesture-handler';
+import {
+  fireGestureHandler,
+  getByGestureTestId,
+} from 'react-native-gesture-handler/jest-utils';
 import WeekView from '../WeekView/WeekView';
 
-it('renders correctly', () => {
-  renderer.create(
-    <WeekView numberOfDays={3} selectedDate={new Date(2021, 3, 24)} />,
-  );
+/**
+ * Extracts the event-id embedded in "WeekViewEvent-<id>".
+ *
+ * Throws an error if the testID does not match the regex.
+ * @param {String} testID
+ * @returns string with event id
+ */
+const extractEventIdFromTestID = (testID) => {
+  const found = testID.match(/WeekViewEvent-(?<eventId>\d+)/);
+  const eventId = found && found.groups && found.groups.eventId;
+  if (!eventId) {
+    throw Error(`Could not find the event id in testID: ${testID}`);
+  }
+  return eventId;
+};
+
+/**
+ * NOTE: In most cases the props provided to <WeekView /> need to be
+ * selected carefully, so that events _are visible in the screen_.
+ * For example:
+ *   - numberOfDays
+ *   - selectedDate (usually, at least the same day the target event's startDate)
+ *   - hoursInDisplay
+ *   - startHour (usually, some hours before the target event)
+ */
+describe('Basic render functionality', () => {
+  it('Renders with no events', () => {
+    // does not throw exception
+    render(
+      <WeekView
+        events={[]}
+        numberOfDays={3}
+        selectedDate={new Date(2020, 3, 24)}
+      />,
+    );
+  });
```

Comment:
I think each test should expect something, can you explain why we should test if it throws an exception or not in this case?

Reply:
> I think each test should expect something

Sounds better, just updated the test

Reply:
It looks good, thanks.